gustaf/helpers/data.py

```diff
@@ -411,32 +413,27 @@ def compute_or_return_saved(*args, **kwargs):
 namedtuple to hold unique information of float type arrays.
 Note that for float types, "close enough" might be a better name than unique.
 This way, all tracked arrays, as long as they are 2D, have a dot separated
-syntax to acces unique info. For example, `mesh.unique_vertices.ids`.
-
-Attributes
------------
-values: (n, d) np.ndarray
-ids: (n) np.ndarray
-inverse: (m) np.ndarray
-intersection: (m) list of list
+syntax to access unique info. For example, `mesh.unique_vertices.ids`.
+"""
+Unique2DFloats.values.__doc__ = "(n, d) np.ndarray"
+Unique2DFloats.ids.__doc__ = "(n, d) np.ndarray"
+Unique2DFloats.inverse.__doc__ = "(n, d) np.ndarray"
+Unique2DFloats.intersection.__doc__ = """(m) list of list
   given original array's index, returns overlapping arrays, including itself.
 """
 
 Unique2DIntegers = namedtuple(
         "Unique2DIntegers", ["values", "ids", "inverse", "counts"]
 )
-"""
-namedtuple to hold unique information of integer type arrays.
+"""namedtuple to hold unique information of integer type arrays.
 Similar approach to Unique2DFloats.
-
-Attributes
------------
-values: (n, d) np.ndarray
-ids: (n) np.ndarray
-inverse: (m) np.ndarray
-counts: (n) np.ndarray
 """
 
+Unique2DIntegers.values.__doc__ = "(n, d) np.ndarray"
+Unique2DIntegers.ids.__doc__ = "(n) np.ndarray"
+Unique2DIntegers.inverse.__doc__ = "(m) np.ndarray"
+Unique2DIntegers.counts.__doc__ = "(n) np.ndarray"
```

Comment:
Can we give a hint, just like default doc, `field number 0` or something? if that gets too complicated, I am very okay with this one